src/main/common/time.h

```diff
@@ -39,8 +39,15 @@ typedef uint32_t timeUs_t;
 
 static inline timeDelta_t cmpTimeUs(timeUs_t a, timeUs_t b) { return (timeDelta_t)(a - b); }
 
+typedef enum {
+    DST_EU,
+    DST_USA,
+} tz_dst_country_e;
+
 typedef struct timeConfig_s {
     int16_t tz_offset; // Offset from UTC in minutes, might be positive or negative
+    bool tz_automatic_dst; // Automatically handle DST or ignore it
+    tz_dst_country_e tz_dst_country;
 } timeConfig_t;
```

Comment:
Rather than 2 variables, use just one uint8_t named `tz_automatic_dst` which can take the values `OFF` (default), `US`,`EU`, etc... That way we use a single byte for storage and users just need to change one setting to make it work. Put a comment at the right indicating that the value comes from `tz_automatic_dst_e`.